js/liquid.js

```diff
@@ -257,14 +257,18 @@ module.exports = class liquid extends Exchange {
             const id = this.safeString (currency, 'currency');
             const code = this.safeCurrencyCode (id);
             const name = this.safeString (currency, 'name');
-            const active = currency['depositable'] && currency['withdrawable'];
+            const depositEnabled = currency['depositable'];
```

Comment:
Let's always use safeValue here, otherwise this line violated the following CONTRIBUTING guideline:

- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#accessing-dictionary-keys